workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java

```diff
@@ -247,6 +247,7 @@ private void getWorksFromWorkDefinition(List<WorkFlowWorkDefinition> workFlowWor
 						return;
 					}
 					responseDTOs.add(WorkDefinitionResponseDTO.fromWorkFlowTaskDefinition(wdt.get()));
+					break;
```

Comment:
this break is not needed at all, n?

Reply:
we need it to avoid the errors (workflow with id not found) when app starts

Reply:
in java, if no break then it continues to the case after it